Won't compile after regex 1.9.0

```
error[E0432]: unresolved import `regex::internal`
  --> /home/runner/.cargo/registry/src/index.crates.io-6f17d22bba15001f/httpmock-0.6.7/src/server/mod.rs:19:12
   |
19 | use regex::internal::Input;
   |            ^^^^^^^^ could not find `internal` in `regex`
```

See: https://github.com/rust-lang/regex/pull/978

--- a/src/server/mod.rs
+++ b/src/server/mod.rs
@@ -16,7 +16,6 @@ use hyper::{
     Body, HeaderMap, Request as HyperRequest, Response as HyperResponse, Result as HyperResult,
     Server, StatusCode,
 };
-use regex::internal::Input;
 use regex::Regex;
 
 use matchers::generic::SingleValueMatcher;

